### ISSUE TITLE
[Dec 2025] Accessibility Audit: Org admin user flows

### ISSUE THREAD

**Issue by DanielSass** [CDCgov/prime-simplereport#9348]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the org admin user flows. Do the page scan as well as any relevant guided tests.
Pages to visit:
* `/app/dashboard`
* `/app/settings/*`

Use cases to test:
* view the analytics dashboard with preset and custom date ranges
* add/edit users
   ⚠ the support admin "Manager users" page shares the same "User information" and "Facility access" tabs as the "Manage users" page for org admins
* create/edit a facility
* view facilities
* manage org
* view patient self-registration settings

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.
Export scan results as a csv and add the file to this ticket.

### Additional context
See tickets from previous audits #8513 #5773, #6333
